### ISSUE TITLE
A small correction for Windows users

### ISSUE THREAD

**Issue by baylf2000** [reactide/reactide#175]
For those users that are not as technically proficient as they could be, it might be worth pointing out that they will need to modify the following line in "package.json" to allow the electron packager to complete. As is, it will attempt to build a package for Mac OS which will fail on Windows machines.

Modify this line as follows:

`    "electron-packager": "electron-packager . --overwrite --platform=win32 --arch=ia32 --prune=true --out=release-builds",`

**Comment by djkarlos** [comment]
Thanks for this. Worked a treat

**Comment by translucentfocus** [comment]
I made a pull-request based on your suggest https://github.com/reactide/reactide/pull/176

**Comment by vdmokstati** [comment]
Could you guys package it into exe and share it? Thanks

**Comment by galbenyosef** [comment]
Hello, would you mind check my issue (and solution)?
https://github.com/reactide/reactide/issues/180
your correction is great but I'm afraid its not enough, thanks in advance

**Comment by baylf2000** [comment]
@galbenyosef I would say they are two completely unrelated issues.

**Comment by translucentfocus** [comment]
@galbenyosef I agree with @baylf2000 we are talking about two completely unrelated issues. @vdmokstati Sorry, but I'm not willing to do so given the current issues with the project
